src/components/containers/derived.js

```diff
@@ -18,8 +18,9 @@ const TraceTypeSection = (props, context) => {
   const {fullContainer} = context;
   if (
     fullContainer &&
-    fullContainer._fullInput &&
-    props.traceTypes.includes(fullContainer._fullInput.type)
+    ((fullContainer._fullInput &&
+      props.traceTypes.includes(fullContainer._fullInput.type)) ||
```

Comment:
@VeraZab `_fullInput` is not available in some code paths, so this is the compromise. Specifically, in the grouped context, the all keys starting with `_` are missing from `container` as per the original logic from the grouped Axes. If I remove this exclusion, we end up with circular JSON objects that cannot be serialized :)